packages/orchestration/src/util/masking.ts

```diff
@@ -0,0 +1,26 @@
+
+import type { DpiConfig } from '../client/api/schema/index.js';
+import type { DpiMaskingConfig } from '../orchestration-types.js';
+
+/**
+ * Convenience function to create masking provider SAP Data Privacy Integration.
+ * @param dpiMaskingConfig - Configuration for the masking provider SAP Data Privacy Integration.
+ * @returns An object with the masking provider configuration.
+ */
+export function buildDpiMaskingProvider(dpiMaskingConfig: DpiMaskingConfig): DpiConfig {
+  return {
```

Comment:
As discussed, would be looking a bit better if you restructure the different properties of the dpiMaskingConfig so that you don't have to write out the entire object's name so many times.